src/domains/community/detail/components/RelatedPosts.jsx

```
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react'
+import { getAllPosts } from '../../service/postService'
+import FreeBoard from '../../../../components/web/FreeBoard'
+import Spinner from '../../../../components/web/Spinner'
+const RelatedPosts = ({ currentId }) => {
```

ESLint

[error] 'currentId' is missing in props validation

(react/prop-types)